evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -86,6 +121,10 @@ pub fn apply(ffi_enabled: bool, call: &HEVMCalls) -> Option<Result<Bytes, Bytes>
             }
         }
         HEVMCalls::GetCode(inner) => get_code(&inner.0),
+        HEVMCalls::EnvBool(inner) => env_bool(&inner.0),
+        HEVMCalls::EnvUint(inner) => env_uint(&inner.0),
+        HEVMCalls::EnvInt(inner) => env_int(&inner.0),
+        HEVMCalls::EnvString(inner) => env_string(&inner.0),
```

Comment:
Could we also add `address`, bytes`, and `bytes32` types?

Reply:
great suggestion, i've added them